src/locales/it/translations/labs/index.ts

```diff
@@ -0,0 +1,29 @@
+export default {
+  labs: {
+    label: 'Laboratori',
+    requests: {
+      label: 'Richieste di laboratorio',
+      new: 'Nuova richiesta di laboratorio',
+      view: 'Visualizza richiesta',
+      cancel: 'Cancella',
```

Comment:
It would be more accurate to translate it into "Annulla"

Reply:
mmm are you sure? In this case 'Cancella' stands for 'Change the state to Cancellato'.
In english is the same as Delete. 

'Annulla' is better when you need to close a modal, or block some operation. What do you think?

Reply:
I don't think it's the same as Delete in this case. It means to cancel a lab request. Imho it should be "Annulla richiesta".

Reply:
Ok, I changed to "Annulla richiesta"